### PR TITLE
add mock styling to simulate agency site

### PR DESCRIPTION
**Pull request by brendansudol**
Making it look more like a real website :)

Screenshot:
image

### CODE REVIEW

**Comment by brendansudol** [comment]
cc: @sashax

File: views/layout.erb [hunk 1]
```diff
@@ -6,9 +6,16 @@
...
 </head>
-<body>
-  <div class="container">
+<body class="bg-darken-1">
+  <div class="h5 px2 py1 bg-navy white center">
+    <img src="/img/usa-flag.png">&nbsp;
+    An official website of the United States Government
+  </div>
+  <div class="py2 mb3 bg-white center">
+    <img src="/img/seal-dhs.svg" class="align-middle" width="100">
```
Comment (sashax) [inline views/layout.erb]: I think this looks great, but I believe we ought not use the DHS seal without permission. I think we need to go all the way and simulate one of their sites or (probably) use another (or no) logo.

Comment (brendansudol) [inline views/layout.erb]: ah, didn't think about that -- i replaced with a mock seal for the time being. thanks for comment.

**Comment by sashax** [comment]
Sweet! LGTM.

One more pester: convention we've followed is to squash all the commits to one for each merge. As soon as you do that, I'll be happy to merge it.

**Comment by brendansudol** [comment]
no probs -- squash complete :)

**Comment by sashax** [comment]
high five!

**Pull request merged**